helpers.lua

```diff
@@ -808,6 +837,16 @@ function edit_user()
     end
 end
 
+-- hash the user password using sha-512 and using {CRYPT} to uses linux auth system
+-- because ldap doesn't support anything stronger than sha1
+function hash_password(password)
+    -- TODO is the password checked by regex? we don't want to
+    -- allow shell injection
+    local mkpasswd  = io.popen("mkpasswd --method=sha-512 '" ..password.."'")
```

Comment:
Does this shows up in `ps -ef` ? Do we care about it ? (Maybe this is the only acceptable solution since we don't have a propre lua lib for that :/)